### PR TITLE
README.rst: Fix getting involved link  

### LINKED ISSUE #128: Fix GETTING INVOLVED link in README.rst
The current link is pointing to old docs i.e. `http://coala.readthedocs.org/en/latest/Getting_Involved/README.html` change it to `https://coala.io/#/getinvolved`
label: difficulty/newcomer
Opened by @aashutoshrathi at [Gitter](https://gitter.im/coala/coala?at=59f84149b20c6424296c2453)

### PR DESCRIPTION
**Pull request by prashant0598**
Fix getting involved link 
This ensures that link pointing to "Getting involved information" page is correct. 
 Closes #128

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [ ] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by prnvdixit** [comment]
@prashant0598 Please finish the tasks that are already assigned to you - https://github.com/coala/corobo/issues/299, https://github.com/coala/coala/issues/4648. Also, these all are difficulty/newcomer tasks - Including this one, this sums it up to 3 newcomer tasks for you. I would recommend to focus on these issues first and then move to higher difficulty issues and leave the newcomer issues for someone who is actually newcomer.

Also, you have made a new PR to the same issue after a long inactivity to the older PR - It is never required. You should **never** make more than one PR to an issue.

Please either unassign yourself from other issues or finish them. In case, any help is needed, you can ask it right away on gitter channel - Someone would be around to help you. :)

**Comment by RaiVaibhav** [comment]
I am making it wip `there are 3 PR` ,it don't need to review, it need to either gets close 
@prashant0598  Its a request if you feel you can solve any of one `PR` totally then close other two,till then I am labeling it wip

**Comment by prashant0598** [comment]
@prnvdixit coala/coala#4648 files could not be found and  coala/corobo#299, has some image problem so i took up this issue and i actually deleted the branch of this duplicate issue so i made a new pr i was inactive due to exams and will now unassign myself from other issues

**Comment by RaiVaibhav** [comment]
@prashant0598  then atleast get unassigned from those maintainers/developers will take care of that issues.

**Comment by prashant0598** [comment]
@RaiVaibhav Done it :+1:

**Comment by RaiVaibhav** [comment]
Its a request, next time please assign issue if you feel you can complete or if you think something unclear in issue then please report.

**Comment by prnvdixit** [comment]
@prashant0598 IMO you needn't unassign yourself from the squirrel image issue - Someone left a review there already, atleast try what he suggested and see if it works 😉

**Comment by prnvdixit** [comment]
Also, I am not sure of this but - You mightn't have deleted your branch as it is still visible in the duplicate PR i.e. "prashant0598-patch-1". In my experience, deletion might have resulted in something like "unknown-repository"...

File: README.rst [hunk 1]
```diff
@@ -61,7 +61,7 @@ GETTING INVOLVED
 
 If you want to contribute to coala-html, please take a look at the `Getting
 Involved Information
-<http://coala.readthedocs.org/en/latest/Getting_Involved/README.html>`__.
+< https://coala.io/#/getinvolved>`__.
```
Comment (jayvdb) [inline README.rst]: why the space before http?

**Comment by prashant0598** [comment]
@prnvdixit  @yashovardhanagrawal Pls review as i hv done required changes

**Comment by yashovardhanagrawal** [comment]
@prashant0598 please insert a line break between the description and `Closes #128`
as I had suggested

**Comment by prashant0598** [comment]
@jayvdb Pls review

**Comment by yashovardhanagrawal** [comment]
ack https://github.com/coala/coala-html/pull/138/commits/f81af086e5d6442ae792db3cd5dc3695a2908253

**Comment by jayvdb** [comment]
@rultor merge

**Pull request merged**